Withings/Constants.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Withings.API.Portable
+{
+    internal class Constants
+    {
+        public const string BaseApiUrl = "https://oauth.withings.com/";
+        public const string TemporaryCredentialsRequestTokenUri = "oauth/request_token";
+        public const string TemporaryCredentialsAccessTokenUri = "oauth/access_token";
+        public const string AuthorizeUri = "oauth/authorize";
+        public const string LogoutAndAuthorizeUri = "oauth/logout_and_authorize";
+
+        public class Headers
+        {
+            public const string XFitbitSubscriberId = "X-Fitbit-Subscriber-Id";
```

Comment:
looks like a fitbit carry-over, maybe put a todo here so we remember to change it later